recipes/streamtracer/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "streamtracer" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamtracer-{{ version }}.tar.gz
+  sha256: 8e70bfde8760564e5e4beadda71c4741d4d0b0e11942d005925a8e11c8cd07fe
+
+build:
+  skip: true  # [py2k or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - benchmarks
```

Comment:
This should not be installed. Typically this means that you need to fix the call to `find_packages`.